src/ruptures/show/display.py

```diff
@@ -69,9 +82,11 @@ def display(signal, true_chg_pts, computed_chg_pts=None, **kwargs):
         for (start, end), col in zip(pairwise(bkps), color_cycle):
             axe.axvspan(max(0, start - 0.5), end - 0.5, facecolor=col, alpha=alpha)
 
-        color = "k"  # color of the lines indicating the computed_chg_pts
-        linewidth = 3  # linewidth of the lines indicating the computed_chg_pts
-        linestyle = "--"  # linestyle of the lines indicating the computed_chg_pts
+        color = (
```

Comment:
Also, can you replace this by
``` 
# color of the lines indicating the computed_chg_pts
color = computed_chg_pts_color
```

The CI spread it on several lines for some reason but this makes the code coverage fail.